### ISSUE TITLE
[iOS] Access to raw image file?

### ISSUE THREAD

**Issue by dog44wgm** [jamesmontemagno/MediaPlugin#892]
I deal with .jpeg files that have thermographic imaging metadata embedded. The MediaPlugin works fine to display and pick the image. Problem: the file I get has been altered in some way so that my thermographic software can no longer load the file. The MediaFile.GetStream function returns a truncated byte array even when PhotoSize is Full and SaveMetaData is True. Is there some way I to avoid this problem? Is it possible to access the original, unaltered file?

**Comment by Alextorres950325** [comment]
yes System.NullReferenceException: Unable to convert image to jpeg, please ensure file exists or lower quality level again appear
